Setup.bat

```diff
@@ -84,12 +87,16 @@ call :MarkStartOfBlock "Retrieve dependencies"
     spatial package retrieve worker_sdk      c-dynamic-x86-msvc_md-win32            %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86-msvc_md-win32.zip"
     spatial package retrieve worker_sdk      c-dynamic-x86_64-msvc_md-win32         %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip"
     spatial package retrieve worker_sdk      c-dynamic-x86_64-gcc_libstdcpp-linux   %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip"
+    spatial package retrieve worker_sdk      core-dynamic-x86_64-linux              %PINNED_CORE_SDK_VERSION%       "%CORE_SDK_DIR%\worker_sdk\core-dynamic-x86_64-linux.zip"

```

Comment:
How come we need two linux libraries? What's the difference between these two?

Reply:
The C# Worker SDK requires the `core-dynamic` package and can't use `c-dynamic`. Checked up with Worker on this, essentially the two packages are the same, but the C# SDK looks for `CoreSdkDll.dll`, which won't be there when using `c-dynamic`

Reply:
Is it possible for the GDK to use the core-dynamic package instead of the c-dynamic one?

Reply:
I'm not sure how/where `c-dynamic` is used, but that might be possible yeah